app/points/calculate-points-for-workload.js

```diff
@@ -1,13 +1,39 @@
 const calculatePointsForTiers = require('../../app/points/calculate-points-for-tiers')
+const calculateSdrConversionPoints = require('../../app/points/calculate-sdr-conversion-points')
+const calculateParomPoints = require('../../app/points/calculate-parom-points')
+const calculateArmsPoints = require('../../app/points/calculate-arms-points')
+
 const Workload = require('../../app/points/domain/workload')
 const CaseTypeWeightings = require('../../app/points/domain/case-type-weightings')
 const assertObjectType = require('../../app/points/domain/validation/assert-object-type')
 
 module.exports = function (workload, caseTypeWeightings) {
```

Comment:
Just need to make sure that the caseTypeWeightings are converted to the correct format in the worker.  Where there any thoughts of bringing the points conversion into probation rules away from the worker?